wordfugue/blog/models.py

```diff
@@ -1,10 +1,96 @@
-from django.db import models
 from django.conf import settings
-from django.urls import reverse
 from django.contrib.sites.models import Site
+from django.db import models
+from django.urls import reverse
 from django.utils import timezone
 from django.utils.html import strip_tags
 from model_utils.models import TimeStampedModel
+from modelcluster.contrib.taggit import ClusterTaggableManager
+from modelcluster.fields import ParentalKey
+from taggit.models import TaggedItemBase
+from wagtail.admin.edit_handlers import FieldPanel, InlinePanel, MultiFieldPanel
+from wagtail.core.fields import RichTextField
+from wagtail.core.models import Orderable, Page
+from wagtail.images.edit_handlers import ImageChooserPanel
+from wagtail.search import index
+
+
+class BlogIndexPage(Page, TimeStampedModel):
```

Comment:
Wagtail has a concept of a combined model + view called a `Page`. So this class controls both the model and the rendering of the model. This particular model, `BlogIndexPage` is a parent page for blog posts to live under.